mmv1/third_party/terraform/utils/privateca_utils.go

```diff
@@ -90,17 +305,26 @@ func flattenPrivatecaCertificateConfigX509ConfigCaOptions(v interface{}, d *sche
 	// and CertificateAuthority APIs.
 	if v == nil || len(v.(map[string]interface{})) == 0 {
 		v = make(map[string]interface{})
-		original := v.(map[string]interface{})
-		transformed := make(map[string]interface{})
-		transformed["is_ca"] = flattenPrivatecaCertificateConfigX509ConfigCaOptionsIsCa(original["isCa"], d, config)
-		return []interface{}{transformed}
 	}
 	original := v.(map[string]interface{})
 	transformed := make(map[string]interface{})
+
+	val, exists := original["isCa"]
 	transformed["is_ca"] =
-		flattenPrivatecaCertificateConfigX509ConfigCaOptionsIsCa(original["isCa"], d, config)
+		flattenPrivatecaCertificateConfigX509ConfigCaOptionsIsCa(val, d, config)
+	// CertificateAuthority does not fields `include_is_ca`.
+	// Expecting non-CertificateAuthority resource does not have field `certificate_authority_id` at top level.
+	if exists && d.Get("certificate_authority_id") == nil {
```

Comment:
@slevenick Do you have any suggestions for another way of checking if `include_is_ca` is present in the schema before setting it?

Reply:
This has been solved by adding `include_is_ca` to the other schema.